test/testdata/infer/generics/bounds.rb

```diff
@@ -1,4 +1,5 @@
 # typed: true
+# disable-fast-path: true
```

Comment:
This is a remaining bug but it's different enough in content that I'd like to tackle it in another PR.

Reply:
…actually, nevermind: it's pretty easy to do now.